crates/syntax_codegen/Cargo.toml

```
@@ -0,0 +1,17 @@
+[package]
+name = "syntax_codegen"
+version = "0.1.0"
```

```suggestion
version = "0.0.0"
```

We wouldn't want to publish it